src/components/footer/footer.component.jsx

```diff
@@ -11,23 +11,25 @@ export const Footer = ({ routes }) => {
     <footer className={classes.footer}>
       <div className={classes.content}>
         <div className={classes.topSection}>
-          <div className={classes.head}>
-            <p>Start your integration</p>
-            <p>Contact us to learn more about Hermez network.</p>
-            <a
-              href='mailto:hello@hermez.io?&subject=Contact%20from%20the%20website'
-              className={classes.emailLink}
-            >hello@hermez.io
-            </a>
+          <div className={classes.col3}>
+            <div className={classes.head}>
+              <p>Start your integration</p>
+              <p>Contact us to learn more about Hermez network.</p>
+              <a
+                href='mailto:hello@hermez.io?&subject=Contact%20from%20the%20website'
+                className={classes.emailLink}
+              >hello@hermez.io
+              </a>
+            </div>
           </div>
-          <div>
+          <div className={classes.col3}>
             <ul className={classes.list}>
               <li className={`${classes.listItem} ${classes.listItemHead}`}>
                 Hermez
               </li>
               <li className={classes.listItem}>
                 <Link to={routes.find(el => el.id === 2).to}>
-                  {routes.find(el => el.label === 'Payments network').label}
+                  {routes.find(el => el.id === 2).label}
```

Comment:
Maybe a comment or a const to explain where the 2 comes from

Reply:
Those are being passed from layout component. Comment added.